### ISSUE TITLE
Record Report (Consultation Details) - Heading and Labels - 3 skipped heading levels on section subheadings

### ISSUE THREAD

**Issue by RobOatesHistoricEngland** [archesproject/Arches_HER#1031]
**Steps to Reproduce:**
Navigate to Record Report - Consultation Details
Enable WAVE browser extension and view heading level alerts.

**Expected** Result:
No skipped heading levels should be present on the page as this hinders the user's ability to orient themselves within the page if they are using a screen reader.

**Actual Result:**
3 skipped heading levels are present on the report section subheadings under 'Consultation Details'.

![image](https://user-images.githubusercontent.com/17125181/195074490-e64ef2aa-3ba1-4b3c-a438-5dc54468ee77.png)

**Ticket Background**

Sponsored by: Historic England
Suggested by: @RobOatesHistoricEngland 
Designed by: @RobOatesHistoricEngland
